lessons/introduction-to-smartpy/03/03.mdx

```diff
@@ -7,12 +7,16 @@ editor:
   startingCode: |
     import smartpy as sp
 
+    # Initialize a python class which extends `sp.Contract` and `pass` it
```

Comment:
Again the same principle, we are handholding users too much, 
 #Initialize a python class is enough (make declarative hint comments rather than imperative)

Reply:
ok